integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/datalake/SdxUpgradeTestAssertion.java

```diff
@@ -29,7 +29,7 @@ public static Assertion<SdxInternalTestDto, SdxClient> validateUnsuccessfulUpgra
         };
     }
 
-    public static Assertion<SdxInternalTestDto, SdxClient> validateSuccessfulUpgrade() {
+    public static Assertion<SdxInternalTestDto, SdxClient> validateUpgradeCandidateWithLockedComponentAvailable() {
```

Comment:
I might rename it to
validateUpgradeCandidateWithLockComponentsSet

Reply:
I don't know why would be that better? what this method does is check if there is an upgrade candidate available when locked component is true. maybe an `Is` before `Available` would help a bit

Reply:
Yes, you are right, I misunderstood the method name. The "Is" would help, indeed.